list-components/Header.js

```diff
@@ -0,0 +1,18 @@
+import React, { useState } from 'react';
+import { StyleSheet, Text, View } from 'react-native';
+
+export default function Header(){
+    return(
+        <View style={styles.header}>
+            <Text>My Todos</Text>
+        </View>
+    )
+}
+
+const styles = StyleSheet.create({
+    header:{
+        height:80,
+        paddingTop:38,
+        backgroundColor:'teal'
+    }
+});
```

Comment:
This file is never used. Is there a reason it needs to be included in the code base?